containers/runner/skip-testtypes

```diff
@@ -10,10 +10,12 @@ common_skip_array=(
 
 rawhide_skip_array=(
   skip-on-fedora
+  rhbz2122327 # installation with an existing DDF RAID device fails
```

Comment:
Please move the disabling to the daily_iso_skip_array. We don't actually keep rawhide_skip_array and rawhide_test_skip_array up to date as we are using only the daily_iso_skip_array. Sorry for that it not very obvious.